https://issues.redhat.com/browse/ACM-10483 (#6291)
open-cluster-management/rhacm-docs: governance/policy_overview.adoc

[#policy-overview]
= Policy overview

> To create and manage policies, gain visibility and remediate configurations to meet standards, use the {product-title} security policy framework. Kubernetes custom resource definition instances are used to create policies. You can create a policy in any namespace on the hub cluster except the cluster namespace. If you create a policy in the cluster namespace, it is deleted by {product-title-short}. Each {product-title-short} policy can have at least one or more templates. For more details about the policy elements, view the _Policy YAML table_ section on this page.

Reviewer: For readability, what do you think of making this a bullet list?

```suggestion
Use the {product-title} security policy framework to help you:
- Create and manage policies
- Gain visibility
- Remediate configurations to meet standards
 Kubernetes custom resource definition instances are used to create policies. You can create a policy in any namespace on the hub cluster except the cluster namespace. If you create a policy in the cluster namespace, it is deleted by {product-title-short}. Each {product-title-short} policy can have at least one or more templates. For more details about the policy elements, view the _Policy YAML table_ section on this page.

```

Author: @jc-berger thanks for your suggestion, I think it will overcrowd the beginning of the file with bullet points. This is how it currently looks after accepting @gparvin suggestion
<img width="1722" alt="Screenshot 2024-03-27 at 1 35 59 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/9ddf3d74-052c-4cb9-a14c-d7a488f8382a">

> To create and manage policies, gain visibility and remediate configurations to meet standards, use the {product-title} security policy framework. Kubernetes custom resource definition instances are used to create policies. You can create a policy in any namespace on the hub cluster except the cluster namespace. If you create a policy in the cluster namespace, it is deleted by {product-title-short}. Each {product-title-short} policy can have at least one or more templates. For more details about the policy elements, view the _Policy YAML table_ section on this page.

Reviewer 2: ```suggestion
To create and manage policies, gain visibility and remediate configurations to meet standards, use the {product-title} security policy framework. Kubernetes custom resource definition instances are used to create policies. You can create a policy in any namespace on the hub cluster except the managed cluster namespaces. If you create a policy in a managed cluster namespace, it is deleted by {product-title-short}. Each {product-title-short} policy can be organized into one or more policy template definitions. For more details about the policy elements, view the _Policy YAML table_ section on this page.
```

> Each client and provider is responsible for ensuring that their managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.

Reviewer 2: ```suggestion
The customer is responsible for ensuring that their managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.
```
"Each client and provider" sounded confusing to me when I read it just now, but it might be just me.

> You are responsible for ensuring that their managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.

Author: ```suggestion
You are responsible for ensuring that the managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.
```

> Each client and provider is responsible for ensuring that their managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.

Author: To maintain active tone
```suggestion
You are responsible for ensuring that their managed cloud environment meets internal enterprise security standards for software engineering, secure engineering, resiliency, security, and regulatory compliance for workloads hosted on Kubernetes clusters.
```

* You must create the `PlacementBinding` to bind your policy with a `Placement` in order to propagate the policy to the managed clusters.
* To use a `Placement` resource, you must bind a `ManagedClusterSet` resource to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to link:../clusters/cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.

> * If you do not select a cluster set, all clusters are selected from all cluster sets that are bound to the namespace. 

Reviewer 2: It could be another entry here.  I think we can simplify it some too since I see we have a Placement section on this information: https://github.com/stolostron/rhacm-docs/blob/e32588c79d04b9340a9b0032a2557b59801d11ea/clusters/cluster_lifecycle/taints_tolerations.adoc#identifying-built-in-taints-to-reflect-the-status-of-managed-clusters

Could we just add: 
* When creating a Placement for your policy in the console, the [placement toleration that reflect managed cluster status](https://github.com/stolostron/rhacm-docs/blob/e32588c79d04b9340a9b0032a2557b59801d11ea/clusters/cluster_lifecycle/taints_tolerations.adoc#identifying-built-in-taints-to-reflect-the-status-of-managed-clusters) are automatically added to the placement resource.
